### ISSUE TITLE
Changing VS removal efficiency of AD not resulting in difference in VS after AD  

### ISSUE THREAD

**Issue by tomhuhh** [RuminantFarmSystems/RuFaS#1540]
Changing VS removal efficiency of AD did not result in difference in VS after AD.   

## Expected Behavior

Difference in VS removal efficiency results in variation in outflow of VS from AD.  

## Current Behavior

By changing the variable `volatile_solids_removal_efficiency_for_treatment` in the manure input file from 0 to 0.45 to 0.9, the VS outflow from AD does not change. 

Efficiency = 0 

![default_ad-degradable-solids-11-Apr-2024_Thu_15-27-27](https://github.com/RuminantFarmSystems/MASM/assets/52800053/00f5ca7e-2bf9-42de-8072-45120ef26118)

Efficiency = 0.45

![default_ad-degradable-solids-11-Apr-2024_Thu_15-24-21](https://github.com/RuminantFarmSystems/MASM/assets/52800053/3bd2c9a7-9534-4d38-976c-791a415691d5)

Efficiency = 0.9 

![default_ad-degradable-solids-11-Apr-2024_Thu_15-40-06](https://github.com/RuminantFarmSystems/MASM/assets/52800053/48d924c4-e93b-44a5-b8a3-d8713d5828ac)

## Steps to Reproduce

1. Go to the main branch, set up the manure scenario to be `5` in the animal input file, and adjust the variable `volatile_solids_removal_efficiency_for_treatment` in the manure input file from 0 to 0.45 to 0.9,
2. Use the graphic filter 

```
{
  "multiple": [
    {
      "type": "plot",
      "filters": [
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_0_CALF\\.liquid_manure_total_degradable_volatile_solids",
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_1_GROWING\\.liquid_manure_total_degradable_volatile_solids",
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_2_CLOSE_UP\\.liquid_manure_total_degradable_volatile_solids",
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_3_LAC_COW\\.liquid_manure_total_degradable_volatile_solids"
      ],
      "title": "AD Degradable Solids",
      "legend": [
        "Pen0",
        "Pen1",
        "Pen2",
        "Pen3"
      ]
    },
    {
      "type": "plot",
      "filters": [
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_0_CALF\\.liquid_manure_total_non_degradable_volatile_solids",
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_1_GROWING\\.liquid_manure_total_non_degradable_volatile_solids",
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_2_CLOSE_UP\\.liquid_manure_total_non_degradable_volatile_solids",
        "AnaerobicDigestion_ManureTreatmentDailyOutput_Pen_3_LAC_COW\\.liquid_manure_total_non_degradable_volatile_solids"
      ],
      "title": "AD Non-Degradable Solids",
      "legend": [
        "Pen0",
        "Pen1",
        "Pen2",
        "Pen3"
      ]
    }
  ]
}
```

4.
5.

**Comment by elle-andreen** [comment]
VS removal is determined based on CH4 production from the digester, it should not be based on a treatment-specific VS removal efficiency (as far as I understand). From what I can recall, I think Kristan wanted all of the treatment removal efficiencies removed at some point, as we have moved toward updating nutrients based on estimated emissions or other losses in order to maintain mass balance through the manure management system. I think a lot of the efficiencies have been set to 0 either in the inputs or default manure treatment configs, or if they're non-zero they are 'overwritten' by other values in the code. Assuming Kristan still wants to go forward with deprecating the treatment removal efficiencies, we can create a PR to remove either just the digestion and lagoon values, or remove them from all treatment inputs & configurations!

**Comment by Isgotkowitz** [comment]
@elle-andreen should this issue be closed as unplanned, because it will be addressed instead by #1803?

**Comment by Isgotkowitz** [comment]
Closed as unplanned - will be ultimately be addressed by #1803
